src/Deposit.php

```diff
@@ -32,20 +32,29 @@ class Deposit
      */
     private $term;
 
+    /**
+     * The Unique Id the deposit.
+     *
+     * @var string
+     */
+    private $unique_id = null;
```

Comment:
This is null by default.

```suggestion
    private $uniqueId;
```